RefRed/version.py

```diff
@@ -3,7 +3,7 @@
 # the minor version and change date gets automatically updated on a git commit
 window_title = 'Liquids Reflectometer Reduction - '
 
-version = (2, 58)
+version = (3, 1)
```

Comment:
`RefRed.__init__.py` contains variable `__version__`. I don't see why do we have to hard-code the version in this file

Reply:
I was recommended to bump this so that when pushed to prod the artifact server would actually update, its a downstream dependence.  Not sure what it would take to use `__version__` instead on their side.

Reply:
It should be automatic. Once we create a tag in the Github repo, the versioneer utility will populate  `__version__` will be the string version specified in the tag.